internal/cmd/login.go

```diff
@@ -143,6 +145,8 @@ func login(cli *CLI, options loginCmdOptions) error {
 	case options.AccessKey != "":
 		loginReq.AccessKey = options.AccessKey
 	case options.User != "":
+		fmt.Fprintf(cli.Stderr, "  Logging in as user %s\n", termenv.String(options.User).Bold().String())
```

Comment:
```suggestion
		fmt.Fprintf(cli.Stderr, "  Logging in as %s\n", termenv.String(options.User).Bold().String())
```

Reply:
This line and the preceding line are very similar so I wanted something more to differentiate between the two